src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -504,10 +506,90 @@
                         <label class="form-check-label" for="reviewViewedSwitch">Mark as Viewed</label>
                     </form>
                 </div>
+            </li>
+            <li class="list-group-item">
+                <div>
+                    <h6>Revision History</h6>
+                        <ul class="approval-list-group" id="approval-list-group">
+                        @foreach (var change in Model.ReviewContent.ActiveAPIRevision.ChangeHistory)
+                        {
+                            <li class="list-group-item approval-list-group-item">
+                                <div class="d-flex align-items-start">
+                                    <div class="me-3">
+                                        @if (change.ChangeAction.ToString() == "Created")
+                                        {
+                                            <i class="bi bi-plus-circle-fill created status-icon"></i>
+                                        }
+                                        else if (change.ChangeAction.ToString() == "Approved")
+                                        {
+                                            <i class="bi bi-check-circle-fill approved status-icon"></i>
+                                        }
+                                        else if (change.ChangeAction.ToString() == "ApprovalReverted")
+                                        {
+                                            <i class="bi bi-arrow-left-circle-fill approval-reverted status-icon"></i>
+                                        }
+                                        else if (change.ChangeAction.ToString() == "Deleted")
+                                        {
+                                            <i class="bi bi-backspace-fill deleted status-icon"></i>
+                                        }
+                                        else if (change.ChangeAction.ToString() == "UnDeleted")
+                                        {
+                                            <i class="bi bi-plus-circle-fill undeleted status-icon"></i>
+                                        }
```

Comment:
What's the default class for empty changeAction? I assume it's going to be empty for existing records.

Reply:
Change action should not be empty though. since each ChangeHistory entry must have indicated an action